File: src/main/java/no/nav/syfo/kafka/producer/model/KTredjepartsvarsel.java
package no.nav.syfo.kafka.producer.model;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDateTime;

@Data
@Accessors(fluent = true, chain = true)
public class KTredjepartsvarsel {
    public String type;
    public String ressursId;
    public String aktorId;
    public String orgnummer;
    public LocalDateTime utsendelsestidspunkt;
}

File: src/main/kotlin/no/nav/syfo/varsel/KTredjepartsvarsel.kt
package no.nav.syfo.varsel

import java.io.Serializable
import java.time.LocalDateTime

data class KTredjepartsvarsel(
    val type: String,
    val ressursId: String,
    val aktorId: String,
    val orgnummer: String,
    val utsendelsestidspunkt: LocalDateTime
) : Serializable
